gradle/tools/detekt/detekt-config.yml

```diff
@@ -68,17 +68,17 @@ comments:
     matchDeclarationsOrder: true
   UndocumentedPublicClass:
     active: true
-    excludes: [ '**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**', '**/samples/kotlin-android-app/**' ]
+    excludes: [ '**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**', '**/samples/kotlin-android-app/**', '**/libs/**' ]
```

Comment:
this is because it has to be public in another lib module? but they don't have public api docs? should they?

Reply:
I will probably add docs in the end